src/Movies.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Poster from './Poster';
+import './Movies.css';
+
+const Movies = (props) => {
```

Comment:
If we have time, it might be worth refactoring the props into a destructured `{ movies }`